src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -49,47 +89,42 @@ const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets, is
         }}
         isRequired
       />
-      {isAdvanced && (
-        <SwitchField
-          label="Use VIP DHCP allocation"
-          name="vipDhcpAllocation"
-          onChange={(checked: boolean) => {
-            let apiVip: string | undefined;
-            let ingressVip: string | undefined;
-            if (checked) {
-              if (initialValues.vipDhcpAllocation) {
-                apiVip = initialValues.apiVip;
-                ingressVip = initialValues.ingressVip;
-              }
-            } else if (!initialValues.vipDhcpAllocation) {
-              apiVip = initialValues.apiVip;
-              ingressVip = initialValues.ingressVip;
-            }
-            setFieldTouched('ingressVip', false);
-            setFieldTouched('apiVip', false);
-            setFieldValue('ingressVip', ingressVip || '');
-            setFieldValue('apiVip', apiVip || '');
-            setTimeout(() => {
-              validateField('ingressVip');
-              validateField('apiVip');
-            }, 0);
-          }}
-        />
+      <SwitchField label="Use VIP DHCP allocation" name="vipDhcpAllocation" />
```

Comment:
```suggestion
      <SwitchField label="Allocate VIPs via DHCP server" name="vipDhcpAllocation" />
```

Good idea to tweak the wording of this too @jtomasek, WDYT of this wording that mentions "server"?